src/jit-format/jit-format.cs

```diff
@@ -181,40 +180,61 @@ private void validate()
                     _syntaxResult.ReportError("Specify --coreclr");
                 }
 
-                // Check that we can find the build log for Windows.
+                // Check that we can find compile_commands.json on windows
                 if (_os == "Windows_NT")
                 {
-                    string logFile = "CoreCLR_Windows_NT__" + _arch + "__" + _build + ".log";
-                    string logFullPath = Path.Combine(_rootPath, "bin", "Logs", logFile);
-                    if (_compileCommands != null)
+                    // If the user didn't specify a compile_commands.json, we need to see if one exists, and if not, create it.
+                    if (!_untidy && _compileCommands == null)
                     {
-                        _buildCompileCommands = false;
-                    }
-                    else if (!_untidy && !File.Exists(logFullPath))
-                    {
-                        _syntaxResult.ReportError("Can't find build log.");
-                    }
-                    else
-                    {
-                        _buildLog = logFullPath;
-                        string[] compileCommandsPath = { _rootPath, "bin", "obj", "Windows_NT." + _arch + "." + _build, "compile_commands.json" };
+                        string[] compileCommandsPath = { _rootPath, "bin", "nmakeobj", "Windows_NT." + _arch + "." + _build, "compile_commands.json" };
                         _compileCommands = Path.Combine(compileCommandsPath);
+                        _rewriteCompileCommands = true;
+
+                        if (!File.Exists(_compileCommands))
```

Comment:
I'm a little concerned about this being stale, but assuming it's not instantaneous to regenerate compile_commands, I'm ok with leaving this for now and seeing how it works.  Doesn't seem likely to cause a problem.